user-manager-service/pom.xml

```diff
@@ -10,10 +10,13 @@
     </parent>
 
     <groupId>com.crm-food-establishment</groupId>
-    <artifactId>user-manager</artifactId>
+    <artifactId>user-auth-service</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <name>user-manager</name>
-    <description>User-manager service, which manages users</description>
+    <name>user-auth-service</name>
+    <description>
+        Provides CRUD operations on Users resources
```

Comment:
remove CRUD from description
We need to keep it universal and consider that in some time we will extend functionality
I prefer having smth like that: 'Service that manages user information including authentication...'

Reply:
new version need to be checked